diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -70,6 +71,31 @@ where
     }
 }
 
+// FIXME: Should we disable select_by when `.group_by` has been called?
```

Comment:
I don't think we need to disable `select_by` when `group_by` has been called, as this is basically the same situation as for normal `select`.